diff --git a/dist/index.d.ts b/dist/index.d.ts
index e6db88c0a4c119f65042814d8592602228a44dc7..003ddf835b0c818309236bd9abd8fd4ada2efb0a 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,4 +1,4 @@
 import styled, { LibraryStyled, Styled, StyledInstance } from './constructors/styled';
 export * from './base';
 export { CSSKeyframes, CSSObject, CSSProp, CSSProperties, CSSPseudos, DataAttributes, DefaultTheme, ExecutionContext, ExecutionProps, FastOmit, IStyledComponent, IStyledComponentFactory, IStyledStatics, Interpolation, PolymorphicComponent, PolymorphicComponentProps, RuleSet, Runtime, StyleFunction, StyledObject, StyledOptions, SupportedHTMLElements, WebTarget, } from './types';
-export { LibraryStyled, Styled, StyledInstance, styled as default, styled };
+export { LibraryStyled, Styled, StyledInstance, styled };
